Gopkg.toml

```diff
@@ -0,0 +1,55 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/Sirupsen/logrus"
+  version = "0.11.5"
+
+[[constraint]]
+  name = "github.com/allegro/marathon-consul"
```

Comment:
I think this should not be here

Reply:
Project contains different packages and these are also dependencies to main program. That's because is also included in Gopkg.

Reply:
Take a [look at other projects](https://github.com/search?o=asc&q=filename%3AGopkg.toml&s=indexed&type=Code&utf8=%E2%9C%93) (e.g., https://github.com/golang/dep/blob/master/Gopkg.toml) you can mark your project as a dependency to itself.